templates/default/config.ini.erb

```diff
@@ -265,82 +284,48 @@ HostName=<%= theMysqld[0] %>
 <% hostId = node['mysql']['id'] -%>
 <% if (node.attribute?(:ndb) && node['ndb'].attribute?(:mysqld) && node['ndb']['mysqld'].attribute?(:private_ips) && !node['ndb']['mysqld']['private_ips'].empty?) -%>
 <% for mysql in node['ndb']['mysqld']['private_ips'] -%>
-# for count in 0..(@num_client_slots-1)
+
+<% for count in 1..@num_ndb_slots_per_mysqld -%>
```

Comment:
@maismail isn't the mysql code duplicated?

Reply:
@SirOibaf nope, i added two attributes (num_ndb_slots_per_client, num_ndb_slots_per_mysqld)to control number of NDB connections for mysql node, and API nodes(namenodes).